src/System.IO.FileSystem/tests/Performance/Perf.Directory.cs

```diff
@@ -58,33 +58,30 @@ public void Exists()
             Directory.Delete(testFile);
         }
 
-        [Benchmark]
-        [InlineData(10, 100)]
-        [InlineData(100, 10)]
-        [InlineData(1000, 1)]
-        [OuterLoop("Takes a lot of time to finish")]
-        public void RecursiveCreateDirectory(int depth, int times)
+        public string GetTestDeepFilePath(int depth)
         {
-            if (!PlatformDetection.IsWindows && depth == 1000)
-                return;
-            
-            // Setup
-            string rootDirectory = GetTestFilePath();
             StringBuilder sb = new StringBuilder(5000);
             for (int i = 0; i < depth; i++)
             {
                 sb.Append(@"\a");
```

Comment:
Missed this originally- you need to use the right separator if you want to run this on Unix.